src/components/Blogs/index.jsx

```diff
@@ -0,0 +1,48 @@
+import React, { useState, useEffect } from "react";
+import SearchBar from "../SearchBar";
+import { Container } from "react-bootstrap";
+import "./index.scss";
+
+export default function Blogs() {
+  const [blogs, setBlogs] = useState([]);
+  const URL =
```

Comment:
maybe smth like BLOGS_URL could be more descriptive?

Reply:
these constants dont go in fn body, move under imports after line 4

Reply:
fixed